src/libs/I13nNode.js

```diff
@@ -150,8 +150,8 @@ I13nNode.prototype.getText = function getText (target) {
     if (!DOMNode && !target) {
         return '';
     }
-    var text = (DOMNode && (DOMNode.value || DOMNode.innerHTML)) ||
-        (target && (target.value || target.innerHTML));
+    var text = (target && (target.value || target.innerHTML)) || 
+        (DOMNode && (DOMNode.value || DOMNode.innerHTML));
```

Comment:
here is for changing the priority of `getText`, when we have a click event and try to get text, target should have higher priority than the original DOMNode, like

```
<I13nDiv>
    foo
   <a>bar</a> // should get `bar` instead of `foo bar`
</I13nDiv>
```